src/Map.js

```diff
@@ -60,9 +60,6 @@ class Map extends React.Component {
         marker_options
       ).bindTooltip(layer => result.location.label);
 
-      // enable marker drawing with options
-      map.pm.enableDraw("Marker", marker_options);
-      map.pm.disableDraw("Marker");
```

Comment:
this is needed per Leaflet.PM docs.... 
```If you are wondering how e.g. the drawPolygon button will enable drawing mode with specific options, here it is: Simply enable drawing mode programatically, pass it your options and disable it again. The options will persist, even when the mode is enabled/disabled via the toolbar.```
https://github.com/codeofsumit/leaflet.pm

Reply:
fixed this and merged to master.

Reply:
In this case, I still don't think it's needed since this is just for drawing the marker and there are no buttons for drawing markers; only the search function will place the marker, and that is still working on my branch.